packages/renderer/src/lib/welcome/WelcomePage.svelte

```diff
@@ -46,13 +48,16 @@ onMount(async () => {
   podmanDesktopVersion = await window.getPodmanDesktopVersion();
 
   onboardingsUnsubscribe = onboardingList.subscribe(value => {
-    // Add "selected" property to each provider and add to onboardingEnabledProviders
-    onboardingProviders = value.map(provider => {
+    // Add "selected" property to each provider
+    const sortedProviders = value.map(provider => {
```

Comment:
Maybe we should not define a custom function sortByPodmanName, and instead create a function that can be used directly with .sort.

```ts
value.map(...).sort(sortByPodmanName)

function sortByPodmanName(a: OnboardingInfoWithSelected, b: OnboardingInfoWithSelected): 1 {
  // handle logic of prioritising podman
  // do not forget to properly sort the others so we have a deterministic sorting
 }
```